### DISCUSSION TITLE
How to stop the network url?

### DISCUSSION THREAD

**Discussion by bhaveshxrawat** [yandeu/five-server#90] [Help / Questions]
Hey, there!

I started trying the Five Server few hours ago, and am happy that it provides the support for network URL out of the box. But, is there any way I can disable it. Because, while I understand that having the network URL allows me to test the app on multiple devices in the dev environment, I can't risk having my file directory exposed on the network. If I could enable the network url when I need and other time I could just use the local URL. That'd be sweet!

Thanks!

**Comment by yandeu** [comment]
It will only be exposed in the local network (home, school, workplace). Your data will not be available over the internet (unless you forward the port on your router).

You could set the host to localhost.
`five-server --host=localhost`

If you use the VSCode extension, there should be a host config in the settings.
